src/test/scala/org/broadinstitute/dsde/firecloud/service/LibraryServiceSpec.scala

```diff
@@ -314,65 +314,38 @@ class LibraryServiceSpec extends FreeSpec with LibraryServiceSupport with Attrib
       }
     }
     "when validating JSON Schema" - {
-      "fails on an empty JsObject" in {
-        val testSchema = FileUtils.readAllTextFromResource("test-attribute-definitions.json")
-        val sampleData = "{}"
-        val ex = intercept[ValidationException] {
-          validateJsonSchema(sampleData, testSchema)
-        }
-        assertResult(13){ex.getViolationCount}
-      }
-      "fails with one missing key" in {
-        val testSchema = FileUtils.readAllTextFromResource("test-attribute-definitions.json")
-        val defaultData = testLibraryMetadata.parseJson.asJsObject
-        val sampleData = defaultData.copy(defaultData.fields-"library:datasetName").compactPrint
-        val ex = intercept[ValidationException] {
-          validateJsonSchema(sampleData, testSchema)
-        }
-        assertResult(1){ex.getViolationCount}
-        assert(ex.getMessage.contains("library:datasetName"))
-      }
-      "fails with two missing keys" in {
-        val testSchema = FileUtils.readAllTextFromResource("test-attribute-definitions.json")
-        val defaultData = testLibraryMetadata.parseJson.asJsObject
-        val sampleData = defaultData.copy(defaultData.fields-"library:datasetName"-"library:datasetOwner").compactPrint
-        val ex = intercept[ValidationException] {
-          validateJsonSchema(sampleData, testSchema)
-        }
-        assertResult(2){ex.getViolationCount}
-      }
```

Comment:
Not your code, but all of the schema tests in this class should be pulled out - they have nothing to do with `LibraryService`. Likewise with a bunch of attribute tests.